python/social_bot/envs/grocery_ground.py

```diff
@@ -262,7 +322,7 @@ def run(self, agent, world):
                         # within 45 degrees of the agent direction
                         hitted_ball = True
                 agent_sentence = yield TeacherAction(
-                    reward=-dist / self._random_range)
+                    reward=-dist / self._random_range + 0.5)
```

Comment:
Add an offset of 0.5, so that the expected return is not negative. Negative reward might cause the agent prefer to get self-killed ('done' and reset) ASAP to get rid of more negative reward.